src/_cffi_src/openssl/ec.py

```diff
@@ -208,7 +213,11 @@
     int nid;
     const char *comment;
 } EC_builtin_curve;
-typedef long point_conversion_form_t;
+typedef enum {
+    POINT_CONVERSION_COMPRESSED = 0,
+    POINT_CONVERSION_UNCOMPRESSED = 1,
+    POINT_CONVERSION_HYBRID = 2,
+} point_conversion_form_t;
```

Comment:
Yep -- these aren't even the real values, but since this for the case where EC is not present we need to have the values present for cffi to extract and then we remove them from the final lib.